### PR TITLE
+LED-Streifen Snackbar

### PR DESCRIPTION
**Pull request by Ranlvor**

### CODE REVIEW

**Comment by Ranlvor** [comment]
Also eigentlich habe ich schon https://github.com/Ranlvor/space-map/commit/f1ffe78a40fcf391283c553be262ac81d3ea94c8 gepusht, aber irgendwas ist gerade langsam bei GitHub

File: config/rzl/index.js [hunk 1]
```diff
@@ -26,6 +26,46 @@ const config: Config = {
...
+        command: {
+          name: "cmnd/tasmota-snackbar/speed",
+          type: types.string
+        },
+        defaultValue: "0"
+      },
+      snackbar_led_online: {
+        state: {
+          name: "tele/tasmota-snackbar/LWT",
+          type: types.string
```
Comment (uwap) [inline config/rzl/index.js]: I think types.options would be more appropriated here?

File: config/rzl/index.js [hunk 2]
```diff
@@ -283,7 +323,51 @@ const config: Config = {
...
+          icon: mdi("settings"),
+          enableCondition: ({ snackbar_led_online }) => snackbar_led_online != "Offline"
+        },
+        {
+          type: "slider",
+          text: "Helligkeit",
+          topic: "snackbar_dimmmer",
+          min: 0,
+          max: 100,
+          icon: mdi("brightness-7"),
```
Comment (uwap) [inline config/rzl/index.js]: Maybe it would be possible to choose the icon dependent on the brightness?

File: config/rzl/index.js (hunk 1 above)
Comment (uwap) [reply config/rzl/index.js]: Especially a mapping to "on" and "off" would be great for consistency.

File: config/rzl/index.js [hunk 3]
```diff
@@ -283,7 +323,51 @@ const config: Config = {
           text: "Snackbar",
           topic: "snackbar",
           icon: mdi("power")
-        }
+        },
```
Comment (uwap) [inline config/rzl/index.js]: Add a new section for the LED strip please.

File: config/rzl/index.js [hunk 4]
```diff
@@ -26,6 +26,46 @@ const config: Config = {
         },
         defaultValue: "off"
       },
+      snackbar_dimmmer: {
```
Comment (uwap) [inline config/rzl/index.js]: Please use CamelCase

**Pull request merged**